tests/config/test_per_directory_config.py

```diff
@@ -0,0 +1,24 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+import tempfile
+from pathlib import Path
+
+from pylint.lint import Run
+
+
+def test_fall_back_on_base_config() -> None:
```

Comment:
```suggestion
def test_fall_back_on_base_config(tmpdir: LocalPath) -> None:
```

There's a pytest fixture for that.